docs/src/tutorial/clibraries.rst

```diff
@@ -502,6 +580,47 @@ instead that accepts an arbitrary Python iterable::
             for value in values:
                 self.append(value)
 
+
+Now we can test our Queue implementation using a python script, e.g.::
+
+    # file: test_queue.py
+
+    from __future__ import absolute_import
+    from __future__ import division
+    from __future__ import print_function
+
+    import queue
+
+
+    Q = queue.Queue()
+
+    Q.append(10)
+    Q.append(20)
+    print(Q.peek())
+    print(Q.pop())
+    print(Q.pop())
+    # print(Q.pop()) # This will raise error.
```

Comment:
"... raise an error."
And, in fact, why not write try-except to show what error that is?